### ISSUE TITLE
Can all images live in a single repo?

### ISSUE THREAD

**Issue by jeffpaul** [10up/wp-local-docker-images#19]
**Describe your question**
We need to figure out if we can do all the images in the existing docker images repo or if we really need to separate them out.

**Comment by eugene-manuilov** [comment]
I am not sure about all images in this repo, but I think that it would be better to keep `wpsnapshots` image in the wpsnaphots repo. It would make the process of releasing new versions and publishing new image tags more convenient.

**Comment by jeffpaul** [comment]
@eugene-manuilov so you're recommending we move https://github.com/10up/wp-local-docker-images/tree/master/wpsnapshots to https://github.com/10up/wpsnapshots?

**Comment by eugene-manuilov** [comment]
@jeffpaul yes, IMO keeping it in the wpsnapshots repo makes more sense to me.

**Comment by eugene-manuilov** [comment]
Created a PR for it: https://github.com/10up/wpsnapshots/pull/72

**Comment by eugene-manuilov** [comment]
72nd has been merged. I have created one more to support develop images: https://github.com/10up/wpsnapshots/pull/73
